src/api/python/z3/z3.py

```diff
@@ -10709,14 +10709,6 @@ def SubSeq(s, offset, length):
     return Extract(s, offset, length)
 
 
-def Strings(names, ctx=None):
```

Comment:
```
z3/z3.py:10712:1: F811 redefinition of unused 'Strings' from line 10694
```

Both implementations are the same but the first is in a better place and the second has a better docstring. So, I took docstring from the second, moved it into the first, and removed the second.